Engine/engine.cpp

```diff
@@ -177,23 +190,25 @@ void Engine::beIdle(bool newlyIdle)
 	_lastRequest = engineState::idle;
 }
 
-
-
 bool Engine::receiveMessages(int timeout)
 {
 	std::string data;
 
 	if (_channel->receive(data, timeout))
 	{
-		if(data =="")
+		if(data == "")
+		{
+			Log::log() << "Received nothing..." << std::endl;
 			return false;
+		}
 
 		// JSONCPP_STRING          err;
```

Comment:
I still need to refactor these into something sensible, but I might not get to them in time. So, I leave this comment in case you want to deal with them in this PR as well! 🙏🏼